content/docs/setup/kubernetes/install/operator/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Operator Install 
+description: Install and configure Istio using the Istio Operator.
+weight: 25
+keywords: [operator,kubernetes,helm]
+---
+
+Follow this guide to install and configure an Istio mesh using an alternate
+installation method: the Istio {{<gloss operator>}}Operator{{</gloss>}}
+installation.
+
+The Istio Operator installation process has several benefits:
```

Comment:
I think in general our docs need an elevator pitch for each of the features, rather than directly in the docs.  This section can probably go somewhere else in the documentation - not directly in this page.

Reply:
Toned down the language to be more consistent with the other install guides.

Reply:
addressed